pkg/plugins/policies/meshtcproute/plugin/v1alpha1/listeners_filterchain.go

```diff
@@ -0,0 +1,19 @@
+package v1alpha1
+
+import (
+	core_xds "github.com/kumahq/kuma/pkg/core/xds"
+	envoy_common "github.com/kumahq/kuma/pkg/xds/envoy"
+	envoy_listeners "github.com/kumahq/kuma/pkg/xds/envoy/listeners"
+)
+
+func buildFilterChain(
```

Comment:
nit: TBH I'd also move this func to `listeners.go`

Reply:
To be honest I like to have it separated, to logically have only `listeners` in `listeners.go` file, especially that this function potentially might grow if we decide to introduce logic to configure `httpConnectionManager` instead of `tcpProxy` if all clusters will be of http-like protocols.